### PR TITLE
[THORN-2225] Create JSONB fraction

### PR DESCRIPTION
**Pull request by sberyozkin**

### CODE REVIEW

**Comment by Ladicek** [comment]
This only provides JAX-RS integration. We also need a separate JSON-B fraction -- similarly to `fractions/javaee/jsonp`.

File: fractions/javaee/jaxrs-jsonb/README.adoc [hunk 1]
```diff
@@ -0,0 +1,4 @@
+= JAX-RS + JSON-B
+
+Provides support within JAX-RS application for JSON Binding processing
```
Comment (Ladicek) [inline fractions/javaee/jaxrs-jsonb/README.adoc]: Make it a single line (1 sentence = 1 line) and probably remove the word "processing".

**Comment by sberyozkin** [comment]
@Ladicek, this is a copy and paste of javaee/jsonp  with few minor updates, JSONP without JAXRS does not make sense...

**Comment by Ladicek** [comment]
But the existing code LGTM.

**Comment by Ladicek** [comment]
@sberyozkin Not really true, you can use JSON-P standalone e.g. with Servlet, same with JSON-B.

**Comment by sberyozkin** [comment]
Or, there is another one... Sorry :-)

**Comment by Ladicek** [comment]
np, thanks :-)

**Comment by sberyozkin** [comment]
@Ladicek, 'jsonb' test is not passing yet at the Servlet level, the module.conf does not make the Yasson implementation visible.

**Comment by sberyozkin** [comment]
never mind, has to add the org.eclipse.yasson module :-)

**Comment by sberyozkin** [comment]
No luck yet though. The servlet does not see it, may it is related to the modules issue ?

Caused by: java.lang.ClassNotFoundException: org.eclipse.yasson.JsonBindingProvider from [Module "deployment.JavaxJsonbTest.war" from Service Module Loader]
	at org.jboss.modules.ModuleClassLoader.findClass(ModuleClassLoader.java:198)

**Comment by Ladicek** [comment]
Maybe, not sure. I _think_ it should be possible to add this fraction to existing `master` branch, so maybe you can test it that way.

**Comment by Ladicek** [comment]
Could you rebase? I think with the latest changes (don't forget about the fraction plugin), the test suite module might even pass :-)

**Comment by sberyozkin** [comment]
I'll create a new branch to clean up the merge and commits history rather than reverting etc

**Pull request closed without merging**